sktime/annotation/base/_base.py

```diff
@@ -92,8 +102,8 @@ def fit(self, X, Y=None):
         Creates fitted model that updates attributes ending in "_". Sets
         _is_fitted flag to True.
         """
-        check_labels(self.labels)
-        check_fmt(self.fmt)
+        check_learning_type(self.learning_type)
```

Comment:
if these are tags, they are not set by the user.
I think we could move this to the test suite, hence. But it also does not harm here, for now.